metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_aspects.py

```diff
@@ -414,12 +406,17 @@ def _create_container_class(
         self, entity: Union[DremioContainer, DremioDataset]
     ) -> Optional[ContainerClass]:
         if entity.path:
-            return ContainerClass(
-                container=self.get_container_urn(
-                    path=entity.path,
-                    name="",
-                )
-            )
+            return ContainerClass(container=self.get_container_urn(path=entity.path))
+        return None
+
+    def _create_container_class_containers(

```

Comment:
I don't believe these containers are required given BrowsePathsV2 entry is using raw strings now.